### PR TITLE
Improve grid layout for statement page

### PR DESCRIPTION
**Pull request by sandriu**
Note: statementMain.tpl should probably be deleted.

### CODE REVIEW

File: templates/bits/image.tpl [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {$condition=$condition|default:$obj->fileExtension}
 {$imgClass=$imgClass|default:''}
 {$spanClass=$spanClass|default:''}
+
+
 {if $obj->fileExtension && $condition}
-  <span class="{$spanClass}">
+  <span class="{$spanClass} person-photo">
```
Comment (CatalinFrancu) [inline templates/bits/image.tpl]: `image.tpl` is used for all kinds of images -- user profile pictures, domain favicons (e.g. the W for links to wikipedia). So `person-photo` may be misleading. Given that it isn't used anywhere, can this be removed?

File: templates/bits/statementMain.tpl [hunk 2]
```diff
@@ -1,36 +1,12 @@
 {$statusInfo=$statement->getStatusInfo()}
```
Comment (CatalinFrancu) [inline templates/bits/statementMain.tpl]: Can you please delete this file in the same pull request, for consistency?

File: www/css/main.css [hunk 3]
```diff
@@ -123,6 +132,14 @@ blockquote {
   flex-basis: 75px;
 }
 
+form {
+outline: 1px solid orange;
+}
+
+.answer-label {
+outline: 1px solid green;
```
Comment (CatalinFrancu) [inline www/css/main.css]: This looks like debug code. :-) It's OK to pull it, but we won't be able to push to production until this is resolved.

File: templates/bits/image.tpl (hunk 1 above)
Comment (sandriu) [reply templates/bits/image.tpl]: You're right. I've corrected this by applying the person-photo class to the function calling image.tpl.

File: templates/bits/statementMain.tpl (hunk 2 above)
Comment (sandriu) [reply templates/bits/statementMain.tpl]: Done.

File: www/css/main.css (hunk 3 above)
Comment (sandriu) [reply www/css/main.css]: Done.

**Pull request merged**